zipline/finance/performance/tracker.py

```diff
@@ -319,10 +319,13 @@ def process_event(self, event):
                 midnight = event.dt
 
             if midnight not in self.all_benchmark_returns.index:
-                raise AssertionError(("Date %s not allocated in  "
-                                      "all_benchmark_returns. Calendar "
-                                      "seems to mismatch with benchmark."
-                                      % midnight))
+                raise AssertionError(
+                    ("Date %s not allocated in all_benchmark_returns. "
+                     "Calendar seems to mismatch with benchmark."
+                     "Benchmark container is=%s" %
```

Comment:
we either want a newline or whitespace between this line and the previous or the line will be harder to read.